README.md

```diff
@@ -1,6 +1,58 @@
+A feature area that indicates the component of the product to which the item belongs. For a list of current product component, see below.
+
+A target release version, indicating the version of the product in which we plan to deliver the feature. Camunda has minor releases twice a year, and alpha releases on a monthly basis. Once we know what version we are delivering a feature, we will update the issue to indicate that information.
+
+Once a feature is delivered, the version label will be applied to the roadmap issue and the issue status will be indicate if the feature was delivered in an alpha or minor version.
+
+Some of our features in "Researching" status may still be in the exploratory stages, and have no timeframe available. These are included in the roadmap only for early feedback. 
+
+### Roadmap phases
+The roadmap is arranged on a project board to give a sense for how far out each item is on the horizon. Every product or feature is added to a particular project board column according to the development phase in which it is. Be sure to read the disclaimer below since the roadmap is subject to change, especially further out on the timeline. 
+
+#### Researching
+We are understanding the problem and a researching solutions. In this phase, we are also looking for feedback from users to confirm our hypothesis.
+
+#### In Progress
+At this stage, we have understood the problem and we are actively developing a solution.
+
+#### Alpha: Primarily for testing and feedback
+Limited availability, requires pre-release agreement. Features still under development, and subject to change. Not recommended for production use, and limited documentation, SLAs or support provided.
+
+#### Minor: Generally available to all customers
+Ready for production use with associated SLA and technical support obligations.
+
+### Public Product Components
+
+- [Zeebe](https://github.com/camunda/zeebe)
+- [Connectors](https://github.com/camunda/connectors)
+- [Camunda Platform 8 Documentation](https://github.com/camunda/camunda-platform-docs)
+- [Camunda Modeler](https://github.com/camunda/camunda-modeler)
+- [Camunda BPMN JS](https://github.com/camunda/camunda-bpmn-js)
+- [Camunda DMN JS](https://github.com/camunda/camunda-dmn-js)
+- [Camunda Automation Platform 7](https://github.com/camunda/camunda-bpm-platform)
+
+### Participation
+
+Anyone can participate in the issues repository to share feedback with Camunda or show support for specific features. 
+
+- *Show support for specific feature*: to show your interest, you can write a comment in an existing issue or simply give a thumb up :+1:
+- *Propose a feature*: Use the issue template to propose a Feature Request. Before you do so, make sure to check the component repository (listed above) to find out if the feaure already exist
+- *Report a bug*: Use the issue template to file a Bug Report. Before you do so, make sure to check the component repository (listed above) to find out if the bug has already been reported
```

Comment:
Let's add a deeplink to the issue template.

Reply:
Let's explain that the should use the respective componet repository to create the issue (if possible / available)